brigade.js

```diff
@@ -0,0 +1,31 @@
+const { events, Job } = require('brigadier');
+
+const coverage = (event, project) => {
+    console.log(`===> Building ${ project.repo.cloneURL } ${ event.commit }`);
+
+    const job = new Job('coverage', 'php:rc-alpine');
+
+    job.tasks = [
+        'set -xe',
+        'apk add --no-cache curl git openssl icu-libs zlib icu-dev zlib-dev $PHPIZE_DEPS',
+        'docker-php-ext-install intl zip',
+        'export COMPOSER_ALLOW_SUPERUSER=1',
+        'curl -s -f -L -o /tmp/installer.php https://getcomposer.org/installer',
+        'curl -s -f -L -o /tmp/install.sig https://composer.github.io/installer.sig',
+        'php -r "\$signature = trim(file_get_contents(\'/tmp/install.sig\')); \$hash = hash(\'SHA384\', file_get_contents(\'/tmp/installer.php\')); if (!hash_equals(\$signature, \$hash)) { unlink(\'/tmp/installer.php\'); echo \'Integrity check failed, installer is either corrupt or worse.\' . PHP_EOL; exit(1); }"',
```

Comment:
Can't we use template strings? Would keep us from escaping the `'` and allow multi lines on complex tasks like this one.